src/Raven.Client/Documents/Session/AbstractDocumentQuery.Includes.cs

```diff
@@ -110,5 +122,20 @@ private void IncludeTimeSeries(string alias, Dictionary<string, HashSet<Abstract
                 }
             }
         }
+        
+        private void IncludeRevisions(DateTime? dateTime)
```

Comment:
This cannot actually accept nullable, so let's remove it.
```suggestion
        private void IncludeRevisions(DateTime dateTime)
```